site.yml

```diff
@@ -0,0 +1,38 @@
+# vim:ft=ansible:
+---
+# Bring up jenkins ci master server.
+- hosts: jenkins-master
+  sudo: yes
+  roles:
+    - geerlingguy.jenkins
+  vars:
+    jenkins_plugins:
+      - git
+      - ssh
+      - job-dsl
+  tasks:
+    - name: install git
+      apt: name=git
+    - name: install maven
```

Comment:
You know what I don't even know why I'm installing it, I thought it may have been a dependency for the job DSL but it's optional so I'll take it out.